src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -69,6 +69,14 @@
                 <default>0</default>
                 <Required>N</Required>
             </suppressVersion>
+            <VisibleEmail type="TextField">
+                <default>localhost</default>
```

Comment:
I guess hostname and email settings are just the other way around for the defaults :)